### DISCUSSION TITLE
Get transmitter and sensor expiry/start date when starting via blujay

### DISCUSSION THREAD

**Discussion by ElyeM** [NightscoutFoundation/xDrip#2829] [General]
When starting a session from xdrip we get the transmitter expiry date in system status.

It would be nice to have that as well when using the bluejay as the main collector and xdrip is only getting the data via Bluetooth.

Basically, when starting a session/transmitter from the bluejay it should send over to xdrip that data as well.

**Comment by Navid200** [comment]
Do you mean xDrip is only receiving from BlueJay and something else is occupying the other  slot of the transmitter?

**Reply by Navid200** [reply]
You don't have to start the session from xDrip to get the transmitter days on the status page as long as xDrip can directly connect to the transmitter even if after the session has started.

**Reply by ElyeM** [reply]
Right,  the medical slot is used up by tislim, run phone collector is off and bluejay uses phone slot.

So xdrip by itself is not connected l, it is only getting data via bluejay.

Xdrip is currently not getting the transmitter days.

I assume the bluejay has it, so my suggestion is that the bluejay should send it over to xdrip and display it there.

**Reply by Navid200** [reply]
I understand your request and see a benefit in adding it.
But, I have a question.
Have you considered using xDrip as the main collector while passing readings to BlueJay?  When/if you walk away from your phone, you can easily switch the BlueJay slot on the watch  so that it becomes an independent collector until you get back to your watch.

**Reply by ElyeM** [reply]
That won't work in my case, as it is a kid who is using the bluejay and she won't be able to switch back and forth settings on the watch.

**Reply by ElyeM** [reply]
Is this something that can be implemented? Can this be transformed into an issue?

**Reply by Navid200** [reply]
I don't believe xDrip shows the transmitter or sensor age if sensor is started using a t:slim pump either.
This cannot be done without adding the ability to BlueJay first.
If BlueJay already had the ability to transmit the info and it was the lack of xDrip ability to show it, we could add a feature request for it.

You can contact BlueJay and see what they think.

https://bluejay.website/contactus

**Comment by Navid200** [comment]
Doesn't the t:slim app show that information?
